app/lib/user-utils.js

```diff
@@ -106,6 +107,16 @@ async function levelChatCreditsString () {
   }
 }
 
+function updateUserCreditsMessage () {
+  globalVar.fetchingCreditsString = true
```

Comment:
shouldn't u be checking that we should not fetch if this value is true?

Reply:
we checked in hintsView and problemAlert View. that code is not worth DRY I think,

Reply:
emmm but yes we can check in the function too. do you think check in function make more sense?